ts/Extensions/DashboardPlugins/HighchartsComponent.ts

```diff
@@ -333,12 +328,12 @@ class HighchartsComponent extends Component<HighchartsComponent.ChartComponentEv
         table.setCell(columnName, rowNumber, valueToSet);
     }
     /**
-     * Handles updating via options
+     * Handles updating via options.
      * @param {Partial<Component.ComponentOptions>} options
```

Comment:
You still have the type (`{Partial<Component.ComponentOptions>}`) in brackets here- please remove it.
Also check other files/part of the code for the same.

Reply:
This is a part of the class. It's fixed in a separate ticket/PR here https://github.com/highcharts/highcharts/pull/18632 that covers classes.